app/dao/broadcast_message_dao.py

```diff
@@ -10,12 +10,8 @@ def dao_get_broadcast_message_by_id_and_service_id(broadcast_message_id, service
     ).one()
 
 
-def dao_get_broadcast_message_by_id(broadcast_message_id):
-    return BroadcastMessage.query.get(broadcast_message_id)
-
-
 def dao_get_broadcast_event_by_id(broadcast_event_id):
-    return BroadcastEvent.query.get(broadcast_event_id)
+    return BroadcastEvent.query.filter(BroadcastEvent.id == broadcast_event_id).one()
```

Comment:
Could you explain this change for me please?

Reply:
`.get` returns `None` if there's no value. `.one()` raises a sqlalchemy `NoResultFound`, so is a bit clearer as to why it failed rather than you being surprised by having a None value later.

(and given that we're getting by ID, we never expect the value not to be present yet anywhere that we call this)